src/components/NewHero/styles/ContactForm.css

```diff
@@ -1,122 +1,119 @@
frtG+xhMYtjCYxTGbZCxiWMYxiWMIBjFsYTGLJgCTAMljAJhEGRInQGhTDCmWwkMJMc3TgTTpW3h9ncXAtaW1FwOo5QKrZLbXJ2kueu1mG7pXSiYa0m6GPGKOwHukHGMNbe4+E5+efTfg/wBCtFukPsz0j+H4zMxBttLFZtSfEmax9XaZelpTVDDCGXaSKRe+4B5xhpp+Ye+XyxPFfpbwvGR2Yo4iiKqr6JvZlHS//UXxDiCOgp0qC01Bvfdif1pXCr+Ye+MNIC2t++SZz9LcLF6lxlWRaeIoirl0V75Wt3+6Lx/FDUQUadMU6YN8o1ud9T75VCd0NVtyMu4zo7GY/PRp0sv9npe++lhpHUeLslBqBXMGUqpv6II27xrKxW/LecUGwG0bhoHDMQaNVals2Unzdr3Uj5xtDG2xBrlL+cz5L7Zr21tyvByzsgl5GgYvGVHdnzMMxJsGawHIeoSxxDijVezbLlemNal9zob2tpsT64vIPlOyCTkaXX4vTqWathgzj+8CQDbqP+5m8RxXatmCKgAACqLacr9THBBIVASQNxYkeO3wjZpFfiGbDph8voMWzX39Ll+17pn5TNP9HkHDd0vI0nh/FciGjUpipTJvlJsQd9DGV+LDs2pUKIpK3pm5ZiOn3eIahblI4eVcuNirZT8byXKLMbfYzBcTRUFGrQWogJIOzAm53t9JGM4wOzahQoikrekb3LDpt0jf0FeZGneNYtsEL3v8JOUOFYDIekWUPSbmIwuUX5fO0y/J+tn7QOL5GsJb6khPTtU2Q9IlkPSeoaknSAaaflk80Xw5PKtTPSLKmemxOFS2YDc/ZlI4YTU9SVi+nYwiDAYGbhwognCiXmcGEQZ02/0QSI5nBIEILEdvOOKAmG1kLK+NHo+Pyim4iolepxBahCr1vM5/Grj7xo8PN8+pGg1Bsd+RmjxFz2LXv6PUW5TCp4gLmBXMGABFyNoGN4uWUpkAuLXuTpPPHe+6eE1/xPVN2ts3gfhPKcHf8UT1dUeafA/CdPTYzM4RhmqslNNyBc8gANSY7E4F13ana5UNnQAkHW1zC8nMaFVqXovVUKK1xddNrHTU945QMbiMRSqLTUKDYAWBtUXYZr6jf0dNTtec+tOm7tT43hXw7KGIYOuZXXVT1APd8xLHDq2dAfvaVfKXiaJQXBlSzoc4Ytfsrk/h3t52nW2hFxcRfk1UvS9c1hNZdM53eLQaqQ7WPJPi0bRWrUJ7MM1tzewHixNhKWJb8Q+C/wDKbWJpqtM5lLpRp0WFIXAqVa2udrakDb1ARl71cesYyeIrWpWLhgDs1wyk9MykiBwnHFmKkk36mX+JgIlZGprTX9CWrUUZgoxBcdnYEmx5d+s8t5O1/wAcDqD85nWsou9416nHtZL/AOan/wDRRC7Reh9sVxMXp2/zU/41noeBYLDVEAdb1POJBLi4zEXAuLjTlOmdy5alc8OMx3Ywi69D7f5Sc69D7f5T2H9A4f8Aw/8AdU+socX4fhqdNrLZ8pKgFydBva+3eZmzOd7amWFutMdgNPAH2gRdBrO9jyX/AJQ2O36q/ARNI+e/gnzm8r+MYwn5VbR2JsNec6qXXfT2H3iN4biFRxmG/Ppz+h9UfxfGBzlspNgcytmAIvoDYX3nP9e7p+9aUle4a/T5iZvDB51X9f5CXkOjeA+IlHhh1qfrn4CW38Z/SfK/xtf0e3ZCqNRzHMDrKhmpw+vnpnD5srf3TyIOuUyvX4TUVM5A03UbgdY19JvvVZ9f0fX8p57yZ9Kt+vN3E1QqXJ/vfKYHks9zWI5uZb8YY/KvUJQXY3JG42F/GT2CnlbvvB7YDzuu6d/XugPihY2Fjy5+J8Zjcb7VKvo+uVyks2uvriyk7en8XH1PkrlYJWWCkErNsK+WdHZJ0Ip9kOkg4deglgUz0k5O6BTbBId1EV/RlO9wlj1E0hTPSEKclWMt+Hr095larwoHYe8zeCSezmeMa5PNUeFujZlXXxjMTUxfJCR3T0QpwuzlkS1jITYea235WjX4xiFXKrPYAgXW5ANvRYi425TZFEDcSHw4Ivac/H/rp5HgcazXJs3eSGmt5P8AEVp0rObG50nomwKndR1gnhNM6GmJrHHTGWW2cmMWqzMp00Hsv9Zq0+NgKA7VabquQVqJW7IPRV1YgG1zYg3iKXBaa3yqVudgz7+F4NXgynTz/wB5/rMXHLe43MsdarF49xMuFpJVrNSXXLWYElrnWy6cxa5POU/Jup/WlHcfgZr1/JsHbN+88q0PJ+rTfPTGtiASSfjLMbsuU109LxBvM/aT+NZ6/glGm+Gph7Gxcg3synO2oI1BnzFaOMFRc6MUzC5GtrbaD1TfFM/lb91tPdJnbMt630YSXHW9dvc5Kw8wVFK/4p/tAOlvRJ7/AHGK4hh0p4erl3KNdybsxtpcneeL7I/lb9x/pCFE/lb91vpM879NeOfaMXjlQ2Jtov8ACJHD8QHLspuPNF/VLFfg9GpYupJIAJ22AE7A8GSjmWm7AMb5TlNjtpp3TplLcZGMbJla3eH8QorTyOCCQQxtve/Ma7GPPFMOFK6sDckZd7+Mwzhf859gkfox/N/tH1knJbw+1XFuy0qhQFmsLL188XlLyeqMUY1FKsXa6kEGbdGmVvrfutaMFSJj1o5zltT7QdY9uJvbL2ptta/z3jS8EtHC/Zzn0zcXgVxCZSdjfQ91pX4PwIYcMEckMb62muzQQZrj1qs8u9wk4Zvze6CcMess3kd5k8cXyUhadhY6wD4SybHSKIm5NRi3d2TbuglY4iRllQgrOj7ToFANCDSAIVpplwaFm0kAQgJFReEDOAhWhXAwgZwEICEMvfUeycTYWggSbQODQgZFpNoEgybyAIVpBEKdadaAQhgwRJEKnNCV4Np1pNGxH3XveQWkWnSonPOzSJ0KktIzSLSMsA80i8i060CDIvJIkWlR14StpaBaCZFM7zFyJ0qOkGdOgQROkyYH/9k=');
```

Comment:
images should be stored as separate files instead of using Base64 encoding, this is because storing in base64 encoding increases file size by ~33%
Use existing `src/assets` or `src/images` for storing the image!